Prac_05/emails.py

```diff
@@ -0,0 +1,33 @@
+def main():
+    """Create dictionary of email to names."""
+    email_to_name = {}
+    email = input("Email: ")
+    while email != "":
+        name = get_name_from_email(email)
+        choice = input(f"Is your name {name}? (Y/n) ").upper()
+        if choice != "Y" and choice != "":
+            name = input("Name: ")
```

Comment:
Maybe add more conditions to allow for more types of input? Like yes/no (see output example on the prac_05 github page)